chxtools/chx_wrapper.py

```diff
@@ -78,25 +81,27 @@ def cw_dscan(mot,mdx,pdx,npoints,acqt='default',pos_ret=True):
     WILL NOT WORK FOR A LIST OF DETECTORS!
     """
     # current detector:
-    acq_pv=session_mgr[ascan.user_detectors[0].name].pvname
-    
+    acq_pv = session_mgr[ascan.user_detectors[0].name].pvname
```

Comment:
`session_mgr` is long-gone too. I guess we should just make a note to revisit this entire file. I suspect it hasn't been used in years, so we need to decide whether to delete it or overhaul it.